scripts/Recovery/GenerateHistory.py

```diff
@@ -0,0 +1,50 @@
+import sys
```

Comment:
Was this file meant to be added as it looks a bit like some example code or a test?